plugins/modules/database/misc/redis_incr.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2021, Andreas Botzner <andreas at botzner dot com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: redis_incr
+short_description: Increment keys in Redis
+description:
+   - Increment integer keys in Redis database by 1 and get new value.
```

Comment:
If the keys must be integer keys, why support float increments?

Reply:
I wanted to add the INCRBYFLOAT command too to this module. Is that a good idea? Or should we have separate modules for each individual Redis command? That would greatly simplify the modules but add more files.